packages/veritone-react-common/src/components/Breadcrumbs/breadcrumbItemShape.js

```diff
@@ -0,0 +1,9 @@
+import { string, bool, func, node } from 'prop-types';
```

Comment:
Why is this in it's own file? Just put it in `BreadcrumbItem.js`

Reply:
Done!